arelle/CntlrCmdLine.py

```diff
@@ -510,7 +518,8 @@ def run(self, options, sourceZipStream=None, responseZipStream=None):
         :param options: OptionParser options from parse_args of main argv arguments (when called from command line) or corresponding arguments from web service (REST) request.
         :type options: optparse.Values
         """
-
+        for b in BETA_FEATURES:
+            setattr(self, b, getattr(options, b))
```

Comment:
Can you do this in the generic cntlr? I'm fine (for now) not creating a GUI menu to enable these, but I don't want feature implementations to have to check if the property exists on the cntrl based on the runtime.

Also, can you make this a `dict[str, bool]` with keys of the feature names so as to avoid any potential conflicts with builtin properties?

Reply:
Roger.
Roger.